packages/rancher-cis-benchmark/charts/templates/patch_default_serviceaccount.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: patch-sa
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded, before-hook-creation, hook-failed
```

Comment:
Curious if we want to delete hook on failure. If it fails it might be good to know why.

Reply:
good point, changed